From 83d5d8a86a72e30e5a42223f781fcb83bcdd0caa Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 18 Sep 2022 18:47:59 +0200
Subject: [PATCH 3/6] debian/copyright: Merge Files paragraphs with same
 license and author

The only difference was the years, but one covers and includes the ones
from the other.
---
 debian/copyright | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/debian/copyright b/debian/copyright
index 6306e34..eef116a 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -87,18 +87,13 @@ License: FHS
  versions, except that this permission notice may be stated in a
  translation approved by the copyright holder.
 
-Files:
- policy/images/policy-install.dia
- policy/images/policy-purge.dia
-Copyright:
- 2005 Margarita Manterola
-License: GPL-2+
-
 Files:
  policy/images/debian-policy-install-conffiles.dia
  policy/images/debian-policy-remove-purge.dia
  policy/images/debian-policy-remove.dia
  policy/images/debian-policy-upgrade.dia*
+ policy/images/policy-install.dia
+ policy/images/policy-purge.dia
 Copyright:
  2005, 2006 Margarita Manterola
 License: GPL-2+
-- 
2.37.2

